src/components/BottomBar/GeoPositionPanel.jsx

```diff
@@ -0,0 +1,250 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import subStateToProps from '../../utils/subStateToProps';
+import Button from '../common/Input/Button/Button';
+import { Icon } from '@iconify/react';
+import Popover from '../common/Popover/Popover';
+import styles from './GeoPositionPanel.scss';
+import Picker from './Picker';
+import Input from '../common/Input/Input/Input';
+import {FilterList, FilterListData } from '../common/FilterList/FilterList'
+import {
+  setPopoverVisibility,
+  reloadPropertyTree
+} from '../../api/Actions';
+import CenteredLabel from '../common/CenteredLabel/CenteredLabel';
+import Dropdown from '../common/DropDown/Dropdown';
+import { useLocalStorageState } from '../../utils/customHooks';
+import AnimatedCheckmark from '../common/AnimatedCheckmark/AnimatedCheckmark';
+
+function MultiStateToggle({title, labels, checked, setChecked}) {
+  return <div className={styles.wrapper}>
+    <p className={`${styles.toggleTitle} ${styles.resultsTitle}`}>Mode</p>
+    <div className={styles.toggles}>
+    {labels.map(label => 
+      <React.Fragment key ={`${label}fragment`}>
+        <input id={label} key={label} className={styles.toggle_option} name="state-d" type="radio" onChange={() => setChecked(label)} checked={ label === checked ? "checked" : ""} />
+        <label htmlFor={label} key={`${label}label`} onClick={() => setChecked(label)}>{label}</label>
+      </React.Fragment>)}
+    <div className={styles.toggle_option_slider}>
+    </div>
+  </div>
```

Comment:
This div should be indented one step (and ofc all the content as well). So it's easy to see which div it corresponds to